scripts/ansible/roles/linux/common/vars/ubuntu/bionic.yml

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+---
+llvm_apt_repository: "deb http://apt.llvm.org/{{ ansible_distribution_release }} llvm-toolchain-{{ ansible_distribution_release }}-7 main"
```

Comment:
Is this right? I thought clang-8 was used on bionic/18.04?

Reply:
This was a copy/paste of an existing file, but yes it should. Thanks!